src/index.ts

```diff
@@ -1,4 +1,5 @@
-const at = require('lodash.at')
+const at = require('lodash/at')
+const isEmpty = require('lodash/isEmpty')
```

Comment:
Why like that? Wouldn't be the same as this?

```
import { at, isEmpty } from 'lodash'
```

Reply:
We need to ensure the tree-shacking works for both cases.